### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected identifier 'RowCaching' (at slick.grid.js:23:11)

### DISCUSSION THREAD

**Discussion by JOSMIG99** [6pac/SlickGrid#856] [Q&A]
Hello everyone, I am migrating an application, I already updated the version of JQuery to v3.7.1, when I updated the slick grid I presented the following error and the comments do not tell me the clear error, I appreciate if you have a solution

![2023-10-02 14_20_47-Window](https://github.com/6pac/SlickGrid/assets/130410116/aa02fcd3-75cb-420a-96c4-67ac14ec11c6)

I also have a doubt if the slick grid source in the repository is in .js given that the project does not support typescript due to its configuration.

**Comment by ghiscoding** [comment]
The code you're showing is from the TypeScript source file (`src/`), but you cannot use that directly it will not work. You most probably did not follow the [Migration Guide 5.0](https://github.com/6pac/SlickGrid/wiki/Major-version-5.0-%E2%80%90-ES6-ESM-and-TypeScript-Support) where it tells you to update your script location and also jQuery is no longer necessary, it was dropped in v4. Please follow the guide to fix your problem
